pypesto/sample/dynesty.py

```diff
@@ -130,7 +138,13 @@ def loglikelihood(self, x):
         if any(x < self.problem.lb) or any(x > self.problem.ub):
             return -np.inf
         # invert sign
-        # TODO this is possibly the posterior if priors are defined
+        if self.objective == "neglogpost":
+            # problem.objective returns negative log-posterior
+            # compute log-likelihood by subtracting log-prior
+            return -1.0 * (
+                self.problem.objective(x) - self.problem.x_priors(x)
```

Comment:
If not already possible, please open an issue so this is easier via e.g. `self.problem.objective(x, priors=False)`, seems like it would be a common thing to check likelihood independently of priors in other contexts